modules/nw-ovn-kubernetes-install-ovnkube-trace-local.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture.adoc
+
+:_content-type: PROCEDURE
+[id="nw-installing-ovnkube-localhost_{context}"]
+= Installing the ovnkube-trace on local host
+
+Follow this procedure to install `ovnkube-trace` on your local host.
```

Comment:
This introduction is self-referential, and it's not in line with the mod docs guidelines.

https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs